### PR TITLE
Support key system in HTMLMediaElement.canPlayType

### PR DESCRIPTION
**Pull request by jasonzhangxx**
b/378908084

### CODE REVIEW

File: cobalt/android/apk/app/src/main/java/dev/cobalt/coat/javabridge/HTMLMediaElementExtension.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  }
+
+  @Override
+  public String getJavaScriptAssetName() {
+    return "html_media_element_extension.js";
+  }
+
+  @CobaltJavaScriptInterface
+  public String canPlayType(String mimeType, String keySystem) {
+    Log.i(TAG, "JA: HTMLMediaElementExtension::canPlayType has been called");
```
Comment (zhongqiliang) [inline cobalt/android/apk/app/src/main/java/dev/cobalt/coat/javabridge/HTMLMediaElementExtension.java:43]: do you still need this log?

Comment (jasonzhangxx) [reply cobalt/android/apk/app/src/main/java/dev/cobalt/coat/javabridge/HTMLMediaElementExtension.java:43]: The pr is still WIP.

File: cobalt/android/apk/app/src/main/java/dev/cobalt/coat/CobaltActivity.java [hunk 2]
```diff
@@ -354,37 +356,46 @@ protected void onCreate(Bundle savedInstanceState) {
...
+              },
+              JAVA_BRIDGE_INITIALIZATION_DELAY_MILLI_SECONDS);
+      return;
     }
 
     // --- Initialize the Java Bridge ---
 
     // 1. Gather all Java objects that need to be exposed to JavaScript.
     javaScriptAndroidObjectList.add(new AmatiDeviceInspector(this));
+    javaScriptAndroidObjectList.add(new HTMLMediaElementExtension(this));
```
Comment (borongc) [inline cobalt/android/apk/app/src/main/java/dev/cobalt/coat/CobaltActivity.java:384]: If we are going to have more Java classes like this, should we consider to make a single Java class that stores all Java objects (and put comments to mention the purpose of the Java object) as a list, and result them here, so we can make the code cleaner?

Comment (jasonzhangxx) [reply cobalt/android/apk/app/src/main/java/dev/cobalt/coat/CobaltActivity.java:384]: I think this should be a temporary solution to add javascript interface.

Comment (jasonzhangxx) [reply cobalt/android/apk/app/src/main/java/dev/cobalt/coat/CobaltActivity.java:385]: Created b/379701165 and added a TODO.

**Pull request merged**